static/src/stylesheets/inline/basher.ThisIsTheNHS.scss

```diff
@@ -0,0 +1,424 @@
+        }
+
+        @include mq(desktop) {
+            margin-left: $gs-gutter;
+            left: 0;
+        }
+
+        @include mq(leftCol) {
+            left: 51px;
+        }
+    }
+
+    .description {
+        position: absolute;
+        top: 0;
+        max-width: gs-span(4);
+        @include fs-header(1);
+        color: #ffffff;
+        margin: 0 $gs-gutter / 2;
+        letter-spacing: -.2px;
+
+        @include mq(mobileLandscape) {
+            margin-left: $gs-gutter;
+        }
+
+        @include mq(tablet) {
+            max-width: gs-span(5);
+            @include fs-header(3);
+        }
+
+        @include mq(desktop, leftCol) {
+            @include fs-header(1);
+            max-width: gs-span(4);
+        }
+    }
+
+    .stripe {
+        position: absolute;
+        z-index: 5;
+        top: 0;
+        height: 60px;
+        width: 100%;
+
+        @include mq(tablet) {
+            height: gs-height(2);
+        }
+    }
+
+    .grid .stripe {
+        z-index: 2;
+    }
+
+    .stripe--dark-blue {
+        background-color: $c-dark-blue;
+        width: gs-span(5) + $gs-gutter;
+        left: gs-span(4) + $gs-gutter;
+
+        @include mq(tablet) {
+            left: gs-span(5) + $gs-gutter;
+        }
+
+        @include mq(desktop) {
+            left: auto;
+            right: -(gs-span(2));
+            width: gs-span(2);
+        }
+    }
+
+    .grid .stripe--dark-blue {
+        right: $gs-gutter;
+        width: gs-span(2);
+
+        @include mq(wide) {
+            width: gs-span(4);
+        }
+    }
+
+    .stripe--blue {
+        background-color: $c-blue;
+        top: 60px + $gs-baseline;
+        right: 40%;
+
+        @include mq(tablet) {
+            width: gs-span(8);
+            top: gs-height(2) + $gs-baseline;
+            right: auto;
+            margin-left: -(gs-span(2));
+        }
+    }
+
+    .stripe--white {
+        background-color: #ffffff;
+        top: 120px + $gs-baseline * 2;
+        left: 20%;
+
+        @include mq(mobileLandscape) {
+            left: 40%;
+        }
+
+        @include mq(tablet) {
+            top: gs-height(4) + $gs-baseline;
+            left: gs-width(3);
+            width: calc(100% - 20px);
+        }
+
+        @include mq(desktop) {
+            left: 0;
+            width: gs-span(6) + $gs-gutter;
+        }
+    }
+
+    .grid .stripe--white {
+        width: gs-span(2);
+        left: $gs-gutter;
+    }
+
+    .line {
+        position: relative;
+        z-index: 10;
+        bottom: -205px;
+        left: 200px;
+        display: none;
+
+        @include mq(tablet) {
+            display: block;
+        }
+
+        @include mq(tablet, desktop) {
+            bottom: -109px;
+            left: 540px;
+        }
+
+        @include mq(leftCol) {
+            left: 175px;
+        }
+
+        @include mq(wide) {
+            left: 96px;
+        }
+    }
+
+    .line__image {
+        position: absolute;
+        top: -99999px;
```

Comment:
Why is this so huge?

IIRC creating large planes will reserve a lot of memory in first-generation iPads.